01_P/P_2_0_02/sketch.js

```diff
@@ -18,22 +18,24 @@
 
 /**
  * drawing with a changing shape by draging the mouse.
- *   
+ *
  * MOUSE
  * position x          : length
  * position y          : thickness and number of lines
  * drag                : draw
- * 
+ *
  * KEYS
  * del, backspace      : erase
  * s                   : save png
  */
+'use strict';
 
 function setup() {
   createCanvas(720, 720);
-  smooth();
   noFill();
   background(255);
+  strokeWeight(2);
```

Comment:
Moved `strokeWeight` and `stroke` to `setup` since they only have to be set once.